test/features/legend.test.ts

```diff
@@ -0,0 +1,41 @@
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS, fakeTextShort } from '../factories';
+
+async function expectEditExpect(tester: any, toContain: string) {
+  expect(tester.text()).toContain(toContain);
+  expect(tester.find('input').length > 0).toBe(false);
+
+  tester.find(`button.btn-edit`).simulate('click');
```

Comment:
Why not `tester.click('button.btn-edit')`? Also, I think you can use regular single quotes around `'button.btn-edit'`.